frontends/parsers/p4/p4parser.ypp

```diff
@@ -207,6 +207,7 @@ inline std::ostream& operator<<(std::ostream& out, const P4::Token& t) {
 %token<Token>      PP "++"
 %token<Token>      DONTCARE "_"
 %token<Token>      MASK "&&&"
+%token<Token>      CONVINIENT_INITIALIZER "..."
```

Comment:
Typo?  There is an English word "convenient", but no English word "convinient".  Not sure if the intent is to be an English word or not, though.